Services/Storage/StorageRecord.cs

```diff
@@ -43,7 +43,7 @@ public static StorageRecord FromDocumentDb(Document document)
             var result = new StorageRecord
             {
                 Id = document.Id,
-                ETag = document.GetPropertyValue<string>("ETag"),
+                ETag = document.ETag,
```

Comment:
I think that I remember that we need line 46 because ETag comes across null, or empty, or something if you read it from the document, and that this will break a test.

Reply:
document.GetPropertyValue<string>("ETag") was null.